conditional/blueprints/housing.py

```diff
@@ -23,8 +26,8 @@ def display_housing():
     log.info('frontend', action='display housing')
 
     # get user data
-
     user_name = request.headers.get('x-webauth-user')
+    account = ldap_get_member(user_name)
```

Comment:
Nitpick: not sure if we've used the language account for this before. But I'd prefer something like current account. That way account can be reserved for some kind of interation